pkg/pillar/types/kubeconfig.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2017-2021 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+type KubeConfig struct {
+	AppInstUUID string                 `json:"appinst-uuid"`
+	Config      map[string]interface{} `json:"kubeconfig"`
```

Comment:
Should this definition include the AppInstMetadataType so that the same publication can be reused for future data flowing from app instance to the controller? 
If so it makes sense renamingh the structs etc to something like AppInstMetaData

Reply:
AppInstMetaData structure is separately defined in zedroutertypes.go